.env

```diff
@@ -0,0 +1,4 @@
+SDK_ENV="local"
+ENV_SDK_URL=""
+ENV_BACKEND_URL=""
+ENV_LOGGING_URL=""
```

Comment:
if possible can we re-use env between demo app & sdk where possible?
for e.g 
```
STATIC_DIR=./dist
HYPERSWITCH_PUBLISHABLE_KEY=
HYPERSWITCH_SECRET_KEY=
HYPERSWITCH_SERVER_URL=
HYPERSWITCH_CLIENT_URL=
```

can we reuse any of this variables which would be serving the same purpose?

Also we don't need to prefix these variables with ENV,
Can you add single line doc-comments describing what these variables do?

Reply:
Updated the README just check once.